app/src/main/java/ch/sheremet/dasfoo/rover/android/client/video/VideoDecoderRunnable.java

```diff
@@ -0,0 +1,110 @@
+package ch.sheremet.dasfoo.rover.android.client.video;
+
+import android.util.Log;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.HttpURLConnection;
+import java.net.MalformedURLException;
+import java.net.URL;
+
+/**
+ * Created by Katarina Sheremet on 6/8/16 1:07 PM.
+ */
+public class VideoDecoderRunnable implements Runnable {
+    private static final String TAG = VideoDecoderRunnable.class.getSimpleName();
+    private String mHost;
+    private String mPassword;
+    private int mPort;
+
+    private HttpURLConnection mUrlConnection = null;
+
+    public VideoDecoderRunnable(final String host, final int port, final String password) {
+        this.mHost = host;
+        this.mPort = port;
+        this.mPassword = password;
+    }
+
+    private void initServerConnection() {
+        try {
+            //TODO:UriBuilder
+            URL url = new URL("https://" + mHost + ":" + mPort);
+            mUrlConnection = (HttpURLConnection) url.openConnection();
+            mUrlConnection.setRequestProperty("X-Capture-Server-PASSWORD",
+                    mPassword);
+            // Todo: change width and height
```

Comment:
Let's stick to a single format of `TODO`, so it is easy to find action items from the code.
My suggestion is `// TODO(username): blah-blah` e.g. `// TODO(ksheremet): change width and height`, WDYT?